bff-tiltaksarrangor/src/main/kotlin/no/nav/amt/tiltak/bff/tiltaksarrangor/TiltakarrangorGjennomforingController.kt

```diff
@@ -157,4 +146,16 @@ class TiltakarrangorGjennomforingController(
 		return gjennomforingIder.filter { allowList.contains(it) }
 	}
 
+	private fun tilgjengeligeGjennomforingIder(ansattId: UUID): List<UUID> {
+		val tilgangTilArrangorIder = arrangorAnsattTilgangService.hentAnsattTilganger(ansattId)
+			.filter { it.roller.contains(ArrangorAnsattRolle.KOORDINATOR) }
+			.map { it.arrangorId }
+
+		val ider = tilgangTilArrangorIder.map {
```

Comment:
kan godt returnere denne direkte isteden for å assigne en variabel

Reply:
Lurer på om denne hører hjemme i en service, men det kan vi evt gjøre senere også